bin/main.dart

```diff
@@ -56,12 +82,33 @@ main(List<String> arguments) async {
 
       print(prettyJson(summary));
     } catch (e, stack) {
-      stderr.writeln("Problem with pkg $pkg ($version)");
-      stderr.writeln(e);
-      stderr.writeln(stack);
+      log.Logger.root.shout("Problem with pkg $pkg ($version)", e, stack);
       exitCode = 1;
     }
   } finally {
     tempDir.deleteSync(recursive: true);
   }
+
+  await subscription.cancel();
+}
+
+void _logWriter(record) {
```

Comment:
nit: always wondered if there is a package for this? let's keep it for now, but I'd really like to know what others are using (and replace my similar hacks with that).